calibration/CTDGVM/CGINS-CTDGVM-09692__20210203.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9692,CC_calibration_date,1612310400000,sensor calibration date (milliseconds since 1970-01-01)
```

Comment:
looks correct, but no calibration file available on vault/alfresco to verify cal date

Reply:
confirmed - looks good